fairmq/tools/runSimpleMQStateMachine.h

```diff
@@ -30,6 +32,15 @@ inline int runStateMachine(TMQDevice& device, FairMQProgOptions& cfg)
         return 0;
     }
 
+    if (cfg.Count("version"))
+    {
+        std::cout << "User device version: " << device.GetVersion() << std::endl;
+        std::cout << "FAIRMQ_INTERFACE_VERSION: " << FAIRMQ_INTERFACE_VERSION << std::endl;
```

Comment:
It is the only FairMQ version we have now, incremented when there are breaking changes. We should replace this with a semantic version at some point, unless we stick to FairRoot version.